Add "Last modified" date to footer area

Herm71 opened the pull request:
## What does this do/fix?

Adds "Last modified:" date to footer area at the bottom of the page

## QA

Links to relevant issues
- #160 

## Tests

Does this have tests?

- [ ] Yes
- [X] No, this doesn't need tests because...
- [ ] No, I need help figuring out how to write the tests.

knice reviewed (changes requested):
Margin is off on last-modified date container.

![screenshot-2024-03-05-164816](https://github.com/ucsc/ucsc-2022/assets/36719/03a1ceb6-11a0-431a-99e2-52ca6edb143e)

Herm71 commented:
I added padding to the Outer div to match Truss. I also adjusted the Media Query to match Truss because the one I used via the Mixin was a different width and when Truss was centered, the LM date was not. Now they're synced.

Brief video:

https://github.com/ucsc/ucsc-2022/assets/1000543/5a0f6a1e-9e71-4a4c-a62d-e62b23aa1d55
